### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by Mr-warriyo** [prakhartiwari0/Arito#20]
**Theme Bugs on Dark Mode :(**
Hey! I think you should add a dark mode theme too on the website, people who use forced dark modes can really find website very ugly compared to light mode..

**To Reproduce**
1. Force your browser to make every site to dark mode.
2. Visit arito site

**Expected behavior**
For dark mode users, it's expected that the site should either automatically change to dark theme or atleast have a button to switch between dark/light modes :)

**Screenshots**

**Smartphone (please complete the following information):**
 - OS: [Android OS]
 - Browser [Firefor Nightly for Developers]
 - Version [Latest]

**Solutions**
1. You can add a button to switch b/w dark & light modes.
2. You can use [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) property of css..

3. THE BEST ONE! add both option 1 & 2.

**Comment by prakhartiwari0** [comment]
Thanks for raising this issue! It is something I didn't thought of.

**Comment by Petrosdevri** [comment]
I would like to help with the dark mode however your background pics need to be changed. Can you develop some dark-mode pics so that we can work with the rest? It will also help me find out which colors to use.

**Comment by prakhartiwari0** [comment]
@Petrosdevri Yeah sure I will be making the necessary images & illustration for dark mode, and then inform you about them!

**Comment by Mr-warriyo** [comment]
If there are specific-dark mode images, maybe I can help you guys with css too! I'm good with color combos :)

**Comment by prakhartiwari0** [comment]
@Mr-warriyo Cool then! You should then share some good color themes for dark mode, you can find the main color theme of the app in basic.css, there are variable declared for them!

**Comment by prakhartiwari0** [comment]
If you can, then use the dev tools to replace the colors of the app with the colors you have decided for dark mode, and then share the app's screenshots here and we will discuss which one looks good. 

We need to remember that this app is having a fun-happy kinda environment, so we shouldn't make it so dark that it looks dull and sad.

**Comment by prakhartiwari0** [comment]
I have been reported multiple times about this issue now!
We need to work on this on high priority!

## Screenshot

![image](https://user-images.githubusercontent.com/65062036/203984578-907316f1-650c-4c64-afef-f787b74b7bb8.png)

**Comment by Petrosdevri** [comment]
These are some of the shades that could be used (I uploaded the image only to show the colors, not the design):
![image](https://user-images.githubusercontent.com/100411628/204139877-f5563f61-0ba1-440b-8ed5-5229b57ea245.png)

:root {
    --dark-grey: #212121;
    --light-dark-grey: #545454;
    --grey: #757575;
    --light-grey: #9e9e9e;
    --gradient: linear-gradient(90deg, var(--light-dark-grey), var(--light-grey));
    --silver: #cfcfcf;
} 

If you want to focus on blue shades instead of dark ones here is an alternative view:
![image](https://user-images.githubusercontent.com/100411628/204140282-2eee8c5c-ee20-4780-a5eb-e65733e35201.png)

:root {
    --dark-blue: #09091b;
    --light-dark-blue: #102a43;
    --blue: #004aad;
    --light-blue: #4d94ff;
    --gradient: linear-gradient(90deg, var(--light-dark-blue), var(--light-blue));
    --very-light-blue: #99c2ff;
}

**Comment by prakhartiwari0** [comment]
Hey @Petrosdevri , I think we first need to have a dark background, and all the text and buttons will be of white color.

**Comment by Petrosdevri** [comment]
Great idea, we can go for it.

**Comment by prakhartiwari0** [comment]
Sure then, I will work on it

**Comment by vinay-devs** [comment]
@prakhartiwari0  I would like to inquire about the availability and assignation of the issue , and if it is currently unassigned, may I proceed to take on the task?

**Comment by prakhartiwari0** [comment]
@vinay-devs Actually what we need to discuss is how to change the background image while switching to dark mode. I think we need to replace the image with SVG code and use hsl() to define the colors. 
If you have any suggestions about how it can be done, please share them, we will be happy to know them.

Happy Contributing!

**Comment by prakhartiwari0** [comment]
@HIMANSHIKSHARMA We need to work on it, currently the need is to decide how to change the light background into a dark one, should we change the image? Should we change the background image's address to a new one or not? That new one will obviously be dark. Or what else can be done?


Or should we drop this feature? I am not sure and need help with all this.

**Comment by maozdemir** [comment]
> @HIMANSHIKSHARMA We need to work on it, currently the need is to decide how to change the light background into a dark one, should we change the image? Should we change the background image's address to a new one or not? That new one will obviously be dark. Or what else can be done?
> 
> Or should we drop this feature? I am not sure and need help with all this.

You can use vectors instead, thus it'll be much easier to change the colour scheme.

**Comment by prakhartiwari0** [comment]
Do you mean Vector Graphics/SVG?

**Comment by maozdemir** [comment]
> Do you mean Vector Graphics/SVG?

exactly.

**Comment by prakhartiwari0** [comment]
@maozdemir Got it, I will look into that and how to implement it :)

**Comment by SrishtiSonam** [comment]
@prakhartiwari0 
I would like to help you with adding a feature for switching mode (light/dark), If you desire or require assistance from me in this matter...

**Comment by oshinsaraf** [comment]
Can you assign me this under GSSOC'23?
